ail_vru_uc1_scenario/docker-compose.yml

```diff
@@ -134,9 +140,11 @@ services:
       && roslaunch carma_carla_agent carma_carla_agent.launch
       role_name:='carma_1'
       host:='172.2.0.2'
-      selected_route:='trb2024_left_turn'
+      selected_route:='vru_uc1_north'
       start_delay_in_seconds:='35'
       spawn_point:='215.21,-169.32,0,0,0,0'"
+    # spawn_point:='258.21,-125.32,0,0,0,90'" for through scenario, north
```

Comment:
Is the idea here to provide a easy place for the tester to switch between scenarios. I would recommend a docker env file for this. This way we do not edit the docker-compose but instead just a list of environment variables in a text file.